src/main/io/gps.h

```diff
@@ -35,6 +35,152 @@
+    CFG_MSGOUT_NMEA_ID_VTG_UART1 = 0x209100b1, // U1
+    CFG_MSGOUT_NMEA_ID_GSV_I2C = 0x209100c4, // U1
+    CFG_MSGOUT_NMEA_ID_GSV_SPI = 0x209100c8, // U1
+    CFG_MSGOUT_NMEA_ID_GSV_UART1 = 0x209100c5, // U1
+    CFG_MSGOUT_NMEA_ID_GLL_I2C = 0x209100c9, // U1
+    CFG_MSGOUT_NMEA_ID_GLL_SPI = 0x209100cd, // U1
+    CFG_MSGOUT_NMEA_ID_GLL_UART1 = 0x209100ca, // U1
+    CFG_MSGOUT_NMEA_ID_GGA_I2C = 0x209100ba, // U1
+    CFG_MSGOUT_NMEA_ID_GGA_SPI = 0x209100be, // U1
+    CFG_MSGOUT_NMEA_ID_GGA_UART1 = 0x209100bb, // U1
+    CFG_MSGOUT_NMEA_ID_GSA_I2C = 0x209100bf, // U1
+    CFG_MSGOUT_NMEA_ID_GSA_SPI = 0x209100c3, // U1
+    CFG_MSGOUT_NMEA_ID_GSA_UART1 = 0x209100c0, // U1
+    CFG_MSGOUT_NMEA_ID_RMC_I2C = 0x209100ab, // U1
+    CFG_MSGOUT_NMEA_ID_RMC_SPI = 0x209100af, // U1
+    CFG_MSGOUT_NMEA_ID_RMC_UART1 = 0x209100ac, // U1
+    CFG_MSGOUT_UBX_NAV_PVT_I2C = 0x20910006, // U1
+    CFG_MSGOUT_UBX_NAV_PVT_SPI = 0x2091000a, // U1
+    CFG_MSGOUT_UBX_NAV_PVT_UART1 = 0x20910007, // U1
+    CFG_MSGOUT_UBX_NAV_SAT_I2C = 0x20910015, // U1
+    CFG_MSGOUT_UBX_NAV_SAT_SPI = 0x20910019, // U1
+    CFG_MSGOUT_UBX_NAV_SAT_UART1 = 0x20910016, // U1
+    CFG_MSGOUT_UBX_NAV_DOP_I2C = 0x20910038, // U1
+    CFG_MSGOUT_UBX_NAV_DOP_SPI = 0x2091003c, // U1
+    CFG_MSGOUT_UBX_NAV_DOP_UART1 = 0x20910039, // U1
+    CFG_SBAS_USE_TESTMODE = 0x10360002, // L
+    CFG_SBAS_USE_RANGING = 0x10360003, // L
+    CFG_SBAS_USE_DIFFCORR = 0x10360004, // L
+    CFG_SBAS_USE_INTEGRITY = 0x10360005, // L
+    CFG_SBAS_PRNSCANMASK = 0x50360006, // X8
+    CFG_SIGNAL_GPS_ENA = 0x1031001f, // L
+    CFG_SIGNAL_SBAS_ENA = 0x10310020, // L
+    CFG_SIGNAL_GAL_ENA = 0x10310021, // L
+    CFG_SIGNAL_BDS_ENA = 0x10310022, // L
+    CFG_SIGNAL_QZSS_ENA = 0x10310024, // L
+    CFG_SIGNAL_GLO_ENA = 0x10310025, // L
+    CFG_PM_OPERATEMODE = 0x20d00001, // E1
+} ubxValgetsetBytes_e;
+
+/*
+ * replaced by macro, to save space; keeping for reference
+typedef enum {
+    SBAS_SEARCH_ALL = 0,
+    SBAS_SEARCH_PRN120,
+    SBAS_SEARCH_PRN121,
+    SBAS_SEARCH_PRN122,
+    SBAS_SEARCH_PRN123,
+    SBAS_SEARCH_PRN124,
+    SBAS_SEARCH_PRN125,
+    SBAS_SEARCH_PRN126,
+    SBAS_SEARCH_PRN127,
+    SBAS_SEARCH_PRN128,
+    SBAS_SEARCH_PRN129,
+    SBAS_SEARCH_PRN130,
+    SBAS_SEARCH_PRN131,
+    SBAS_SEARCH_PRN132,
+    SBAS_SEARCH_PRN133,
+    SBAS_SEARCH_PRN134,
+    SBAS_SEARCH_PRN135,
+    SBAS_SEARCH_PRN136,
+    SBAS_SEARCH_PRN137,
+    SBAS_SEARCH_PRN138,
+    SBAS_SEARCH_PRN139,
+    SBAS_SEARCH_PRN140,
+    SBAS_SEARCH_PRN141,
+    SBAS_SEARCH_PRN142,
+    SBAS_SEARCH_PRN143,
+    SBAS_SEARCH_PRN144,
+    SBAS_SEARCH_PRN145,
+    SBAS_SEARCH_PRN146,
+    SBAS_SEARCH_PRN147,
+    SBAS_SEARCH_PRN148,
+    SBAS_SEARCH_PRN149,
+    SBAS_SEARCH_PRN150,
+    SBAS_SEARCH_PRN151,
+    SBAS_SEARCH_PRN152,
+    SBAS_SEARCH_PRN153,
+    SBAS_SEARCH_PRN154,
+    SBAS_SEARCH_PRN155,
+    SBAS_SEARCH_PRN156,
+    SBAS_SEARCH_PRN157,
+    SBAS_SEARCH_PRN158,
+} ubxSbasPrnScan_e;
+*/
+
```

Comment:
This is unneeded and very long. A short descriptive comment above the macro is better

Reply:
Made smaller